data/content_scripts/privly.js

```diff
@@ -0,0 +1,985 @@
+                same = ! privly.correctIndirection.testAndCopyOver(a, attrib.value);
+
+                // The attribute has the injection parameter but it did not pass
+                // replacement. In many cases this indicates the URL will be
+                // swapped in by the host page's JS when the element is hovered
+                // so we hover the element then run it through this process again.
+                if ( same && a.getAttribute("data-privly-hovered") !== "true") {
+                  a.setAttribute("data-privly-hovered", "true");
+                  privly.correctIndirection.hover(a);
+                  var rem = privly.correctIndirection.moveFromAttributes([a]);
+                  rem = privly.correctIndirection.moveFromBody(rem);
+                  same = (rem.length === 1);
+                }
+              }
+            }
+          }
+          if ( same ) {notUpdated.push(a);}
+        }
+      );
+      return notUpdated;
+    },
+
+    /**
+     * Copy matching links from the displayed text of the link.
+     * @param {array} elements array of "a" elements.
+     * @return {array} Elements not updated by the function.
+     */
+    moveFromBody: function(elements) {
+      "use strict";
+      var notUpdated = [];
+      elements.forEach(
+        function(a){
+          // Optimization
+          if (  a.textContent.indexOf("privlyInject1") <= 0 
+            || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
+            notUpdated.push(a);
+          }
+      });
+      return notUpdated;
+    },
+
+
+    /**
+     * Process all the links on the page to discover their true linked
+     * destination. The destination is stored to a new attribute on the
+     * link, `data-privlyHref`.
+     */
+    run: function() {
+      "use strict";
+      var anchors = document.links;
+      var remaining = [];
+      var i = 0;
+
+      // Pre-process
+      for( ; i < anchors.length; i++ ) {
+        var a = anchors[i];
+
+        // Save the current href
+        a.setAttribute("data-privlyHref", a.href);
+
+        // Don't correct further indirection if the href passes.
+        // Some sites shorten the displayed link while giving the
+        // actual link in the anchor.
+        privly.privlyReferencesRegex.lastIndex = 0;
+        if (a.href && !privly.privlyReferencesRegex.test(a.href))
+        {
+          remaining.push(a);
+        }
+      }
+
+      //check if Privly was moved to another attribute
+      remaining = privly.correctIndirection.moveFromAttributes(remaining);
+
+      //check if Privly is in the body of the text
+      privly.correctIndirection.moveFromBody(remaining);
+    }
+  },
+
+  /**
+   * Counter for injected iframe identifiers. This variable is also used
+   * to indicate how many iframes have been injected so that the script
+   * will not inject too many iframes.
+   */
+  nextAvailableFrameID: 0,
+
+
+  injectLinkApplication: function(object, applicationUrl, id)
```

Comment:
Looks like you forgot to copy the comment block for this function.

https://github.com/privly/privly-firefox/blob/master/chrome/content/privly.js#L263